.github/workflows/ci.yml

```
@@ -0,0 +1,54 @@
+  # TURSO_CONNECTION_URL: ${{ secrets.TURSO_CONNECTION_URL }}
+  # TURSO_AUTH_TOKEN: ${{ secrets.TURSO_AUTH_TOKEN }}
+  # JWT_SECRET: ${{ secrets.JWT_SECRET }}
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)